packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/update_pin_page.dart

```diff
@@ -0,0 +1,143 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../app_extensions.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../bloc/update_and_verify_pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+import '../services/update_and_verify_pin_code_service.dart';
+
+class UpdatePinPage extends StatefulWidget {
```

Comment:
- After I update the PIN (current, new, confirm -> success) and return to the Profile page, a screen requiring PIN pops out. Can you please check it out?

Reply:
For the first bullet , this should be the VerifyPinCodePage, where the app navigates after some seconds of inactivity. For the second bullet I have tried some time time to use one page and update it accordingly , but until I separated the logic in different pages, the feature was not working.